Caculator_G/Caculator_G/ViewController.swift

```diff
@@ -11,21 +11,85 @@ import UIKit
 class ViewController: UIViewController {
     
     @IBOutlet weak var numLabel: UILabel!
+    var labelFontSize : CGFloat = 70
+    var numValueText : String = "0"
+    var numDisplayText : String = "0"
+    var isNegative : Bool = false
```

Comment:
```suggestion
    @IBOutlet private weak var numLabel: UILabel!
    private var labelFontSize : CGFloat = 70
    private var numValueText : String = "0"
    var numDisplayText : String = "0"
    var isNegative : Bool = false
```

class 내에서만 사용되는 property는 접근제한자를 설정해주면 이 class에서만 사용된다는걸 알아채기 쉽답니다 !